src/components/ProblemsTable.tsx

```diff
@@ -29,6 +30,7 @@ type Report = {
   type?: string | undefined;
   fileName?: string | undefined;
   fileId: string | undefined;
+  issueid?: string | undefined;
```

Comment:
camel case is used in this project. Could you please update it to be consistent? issued -> issueId